.github/workflows/release.yml

```
@@ -86,7 +86,7 @@ jobs:
         uses: dawidd6/action-get-tag@v1.1.0
       - name: Get release text
         id: tag_text
-        uses: ericcornelissen/git-tag-annotation-action@v1.1.5
+        uses: ericcornelissen/git-tag-annotation-action@v2
```

Should stick with pinned versions here.

```suggestion
        uses: ericcornelissen/git-tag-annotation-action@v2.0.0
```